kolibri/auth/api.py

```diff
@@ -225,6 +227,8 @@ def retrieve(self, request, pk=None):
         return Response(self.get_session(request))
 
     def get_session(self, request):
+        # Set last activity on session to the current time to prevent session timeout
+        request.session['last_activity'] = int(time.time())
```

Comment:
This is more like the last recorded API interaction, but not necessarily overall activity? It might also sit well in middleware?

Reply:
I could have used the `SAVE_SESSION_EVERY_REQUEST` Django setting to achieve this, but I was a little concerned about doing a write to the database on every request, due to performance issues.

This has the nice effect of doing this at a frequent enough interval that our frontend heartbeat code will keep it alive, while not having to do it on every request.

Reply:
This is also the same approach that this library takes too: https://github.com/tzulberti/django-session-idle-timeout

Reply:
Sounds good, maybe rename the session to `last_api_call` or something?

Reply:
Updated.